pages/api/twilio.ts

```diff
@@ -1,25 +1,22 @@
 import { NextApiRequest, NextApiResponse } from 'next'
 import twilio from 'twilio'
 
-import { validatePhoneNumber, INVALID_NUMBER } from '../../helpers/validators'
+import {
+  validatePhoneNumber,
+  INVALID_NUMBER,
+  validateRequest,
+  POST,
+} from '../../helpers/validators'
```

Comment:
added another validator for the request. I thought this was nice

Reply:
thanks!